docs/readthedocs/source/doc/PPML/Overview/misc.rst

```diff
@@ -2,15 +2,14 @@ Advanced Topic
 ====================
 
 
-* `Privacy Preserving Machine Learning (PPML) User Guide <ppml.html>`_
+* `Develop your own Big Data & AI applications with BigDL PPML <devguide.html>`_
 * `Trusted Big Data Analytics and ML <trusted_big_data_analytics_and_ml.html>`_
 * `Trusted FL (Federated Learning) <trusted_fl.html>`_
 * `Secure Your Services <../QuickStart/secure_your_services.html>`_
-* `Deploy PPML (Privacy Preserving Machine Learning) Applications in the Production Environment <../QuickStart/deploy_ppml_in_production.html>`_
-* `Install SGX Driver through the Installation Package <../QuickStart/install_sgx_driver.html>`_
+* `Deploy PPML Applications in the Production Environment <../QuickStart/deploy_ppml_in_production.html>`_
+* `Install SGX Driver for Xeon Server <../QuickStart/install_sgx_driver.html>`_
 * `Deploy the Intel SGX Device Plugin for Kubernetes <../QuickStart/deploy_intel_sgx_device_plugin_for_kubernetes.html>`_
 * `Trusted Cluster Serving with Graphene on Kubernetes <../QuickStart/trusted-serving-on-k8s-guide.html>`_
 * `TPC-H with Trusted SparkSQL on Kubernetes <../QuickStart/tpc-h_with_sparksql_on_k8s.html>`_
 * `TPC-DS with Trusted SparkSQL on Kubernetes <../QuickStart/tpc-ds_with_sparksql_on_k8s.html>`_
-* `Privacy Preserving Machine Learning (PPML) on Azure User Guide <azure_ppml.html>`_
-* `BigDL-PPML-Azure-Occlum-Example <azure_ppml_occlum.html>`_
\ No newline at end of file
+* `PPML on Azure with Occlum <azure_ppml_occlum.html>`_
```

Comment:
From my side, the list of hyperlinks here should have the same contents and order as the side left bar for clear navigation purposes. But they are currently not the same. Should we reorg the links here to make them the same as the following lists?
  - file: doc/PPML/Overview/ppml
  - file: doc/PPML/Overview/devguide
  - file: doc/PPML/Overview/trusted_big_data_analytics_and_ml
  - file: doc/PPML/Overview/trusted_fl
  - file: doc/PPML/QuickStart/secure_your_services
  - file: doc/PPML/QuickStart/install_sgx_driver
  - file: doc/PPML/QuickStart/deploy_intel_sgx_device_plugin_for_kubernetes
  - file: doc/PPML/QuickStart/trusted-serving-on-k8s-guide
  - file: doc/PPML/QuickStart/tpc-h_with_sparksql_on_k8s
  - file: doc/PPML/QuickStart/tpc-ds_with_sparksql_on_k8s
  - file: doc/PPML/QuickStart/deploy_ppml_in_production
  - file: doc/PPML/Overview/azure_ppml_occlum

Reply:
Tks. Now they are the same.